core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -152,10 +152,12 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
 
   /**
    * Compute a histogram using the provided buckets. The buckets are all open
-   * to the right except for the last which is closed
+   * to the right except for the last which is closed.
+   * {{{
    *  e.g. for the array
    *  [1, 10, 20, 50] the buckets are [1, 10) [10, 20) [20, 50]
    *  e.g 1<=x<10 , 10<=x<20, 20<=x<=50
+   * }}}
```

Comment:
Yeap. I tested some cases with it (If I remember correctly) and it seems any `{@foo }` syntax is treated as back-tick-quoted by scaladoc (I will test more in case and will be back here to fix the comment if I was wrong). Sure, sounds great!